distribution/lib/Standard/Tableau/0.0.0-dev/THIRD-PARTY/NOTICE

```diff
@@ -5,8 +5,3 @@ Copyright 2020 - 2025 New Byte Order sp. z o. o.
 The license file can be found at `licenses/APACHE2.0`.
 Copyright notices related to this dependency can be found in the directory `net.java.dev.jna.jna-5.14.0`.
 
-
-'jna-platform', licensed under the Apache-2.0, is distributed with the Tableau.
```

Comment:
- dependency on `jna-platform` is gone
- transitive dependency on `net.java.dev.jna.jna-5.14.0` remains